pubsub/gossipsub/README.md

```diff
@@ -41,7 +41,7 @@ profiles.
 
 - Go: [libp2p/go-libp2p-pubsub/gossipsub.go](https://github.com/libp2p/go-libp2p-pubsub/blob/master/gossipsub.go) (experimental)
 - JS: [ChainSafeSystems/gossipsub-js](https://github.com/ChainSafeSystems/gossipsub-js) – work in progress; check branches and PRs.
-- Rust: [libp2p/rust-libp2p#898](https://github.com/libp2p/rust-libp2p/pull/898), which implements this spec, although missing some features, and [libp2p/rust-libp2p#767](https://github.com/libp2p/rust-libp2p/pull/767), which is a partial implementation, and differs in some ways to this spec—see [#142](https://github.com/libp2p/specs/issues/142) ofr details.)
+- Rust: [libp2p/rust-libp2p#898](https://github.com/libp2p/rust-libp2p/pull/898), which implements this spec, although is missing some features, and [libp2p/rust-libp2p#767](https://github.com/libp2p/rust-libp2p/pull/767), which is a partial implementation, and differs in some ways to this spec—see [#142](https://github.com/libp2p/specs/issues/142) for details.)
```

Comment:
Wow that's a mouthful... I'm not sure that "is" really belongs either.

Maybe:

```
Rust: [libp2p/rust-libp2p#898](https://github.com/libp2p/rust-libp2p/pull/898) implements the spec but is missing some features. [libp2p/rust-libp2p#767](https://github.com/libp2p/rust-libp2p/pull/767) is an alternative, partial implementation that differs from the spec (see [#142](https://github.com/libp2p/specs/issues/142) for details).
```